lib/system/web/src/keyboard_engine.rs

```diff
@@ -0,0 +1,122 @@
+use std::collections::{HashMap, BTreeSet};
+
+use web_sys::{KeyboardEvent};
+use wasm_bindgen::JsCast;
+use wasm_bindgen::prelude::*;
+
+use crate::*;
+
+type Bindings = HashMap<BTreeSet<u32>, Vec<Box<dyn FnMut()>>>;
```

Comment:
I dont understand one thing here. We dont have keyboards with more than 110 keys. In fact, every keyboard in the world contains less than 128 keys (am I right here, or am I missing some strange cases?). If Im right, then instead of `BTreeSet<u32>` we can just keep here `u128` as a bitfield, which will be much faster to compare and work on.

Reply:
Do you suggest me to map every existed key into some bit position by my own? But the mapping table needs to support. Do you see any way to map keyCode/key into the bit position